main.tf

```diff
@@ -57,6 +57,36 @@ resource "aws_iam_service_linked_role" "default" {
   description      = "AWSServiceRoleForAmazonElasticsearchService Service-Linked Role"
 }
 
+# Role that pods can assume for access to elasticsearch and kibana
+resource "aws_iam_role" "elasticsearch_user" {
+  count              = "${var.enabled == "true" ? 1 : 0}"
+  name               = "${module.label.id}-user"
```

Comment:
Use `label` module

Reply:
I'm using the `label` module to generate the base label. It seems excessive to use a second label module just to tack on `-user`. Why do you think it is worth it?